and-bible/app/src/main/java/net/bible/android/view/activity/base/IntentHelper.java

```diff
@@ -22,7 +22,7 @@
 public class IntentHelper {
 
 	// request codes passed to and returned from sub-activities
-	public static final int REFRESH_DISPLAY_ON_FINISH = 2;
+	public static final int RETURN_FROM_SETTINGS_ACTIVITY = 2;
```

Comment:
I don't understand the reason for this rename.  Various screens can require a screen refresh when the user returns from them, not just the settings screen.